erigon-lib/chain/chain_config.go

```diff
@@ -625,11 +639,11 @@ func asSprints(configSprints map[string]uint64) sprints {
 // Rules is a one time interface meaning that it shouldn't be used in between transition
 // phases.
 type Rules struct {
-	ChainID                                                 *big.Int
-	IsHomestead, IsTangerineWhistle, IsSpuriousDragon       bool
-	IsByzantium, IsConstantinople, IsPetersburg, IsIstanbul bool
-	IsBerlin, IsLondon, IsShanghai, IsCancun, IsPrague      bool
-	IsEip1559FeeCollector, IsAura                           bool
+	ChainID                                                           *big.Int
+	IsHomestead, IsTangerineWhistle, IsSpuriousDragon                 bool
+	IsByzantium, IsConstantinople, IsPetersburg, IsIstanbul           bool
+	IsBerlin, IsLondon, IsShanghai, IsBorShanghai, IsCancun, IsPrague bool
```

Comment:
Do we keep `BorShanghai` or have we considered to call it `Agra`?

Reply:
Working on changing this